Nos/Assets/Localization/ImagePicker.xcstrings

```diff
@@ -176,6 +176,39 @@
         }
       }
     },
+    "getAccount" : {
+        "extractionState" : "manual",
+        "localizations" : {
+          "en" : {
+            "stringUnit" : {
+              "state" : "translated",
+              "value" : "Get Account"
+            }
+          }
+        }
+      },
+      "errorUploadingFileExceedsSizeLimit" : {
+            "extractionState" : "manual",
+            "localizations" : {
+              "en" : {
+                "stringUnit" : {
+                  "state" : "translated",
+                  "value" : "Get a Nostr.Build account to upload larger files"
```

Comment:
```suggestion
                  "value" : "Get a Nostr.build account to upload larger files"
```

Reply:
Done. 

Thanks for the lead on the FileStorageAPIClientError, that's a much better place to put the logic rather than in the view. Finding where things go in swift is just different than other environments. 

One thing with swiftlint... there are some catch -22 situations where it complains about it being a long line, but if i break it up then the code works, but it complains i broke up and didn't put it all on a single line. Is 120 characters maybe a bit too low of a limit?